tests/test_autotuner.py

```diff
@@ -238,8 +238,7 @@ def test_multiple_blocking():
                   opt=('blocking', {'openmp': False}))
 
     # First of all, make sure there are indeed two different loop nests
-    assert 'bf0' in op._func_table
-    assert 'bf1' in op._func_table
+    _, _ = assert_blocking(op, {'x0_blk0', 'x1_blk0'})
```

Comment:
just `assert_blocking(op, {'x0_blk0', 'x1_blk0'})` no need to catch the return value if ever used!

replicate this change wherever needed

Reply:
ok